simulators/ethereum/engine/suites/cancun/tests.go

```diff
@@ -587,13 +587,12 @@ var Tests = []test.Spec{
 		BaseSpec: test.BaseSpec{
 			Name: "ForkchoiceUpdatedV2 To Request Shanghai Payload, Zero Beacon Root",
 			About: `
-			Test sending ForkchoiceUpdatedV2 to request a Cancun payload:
-			- Payload Attributes uses Shanghai timestamp
-			- Payload Attributes' Beacon Root zero
+			Test sending ForkchoiceUpdatedV2 to request a Shanghai payload:
+			- Payload Attributes uses Shanghai timestamp with Beacon Root
 
 			Verify that client returns INVALID_PARAMS_ERROR.
 			`,
-			MainFork:   config.Cancun,
+			MainFork:   config.Shanghai,
 			ForkHeight: 1,
```

Comment:
```suggestion
			MainFork:   config.Cancun,
			ForkHeight: 2,
```
I think this would have the same effect, and we also have to remove the `DowngradeForkchoiceUpdatedVersion`, otherwise we might end up sending `engine_forkchoiceUpdatedV1`